examples/aliens.py

```diff
@@ -244,7 +244,7 @@ def main(winstyle=0):
     pg.init()
     if pg.mixer and not pg.mixer.get_init():
         print("Warning, no sound")
-        pg.mixer = None
+        # pg.mixer = None
     pg.mixer = None
```

Comment:
How can the outcome be different if `mixer` is still assigned to `None` when leaving the if clause?